language-support/codegen-main/BUILD.bazel

```diff
@@ -36,3 +39,16 @@ pom_file(
     target = ":shaded_binary",
     visibility = ["//visibility:public"],
 )
+
+# Create empty Scaladoc JAR for uploading to Maven Central
+scaladoc_jar(
```

Comment:
Should we make `scaladoc_jar` and `scala_source_jar` implicit in `da_scala_binary`? Having to add that manually everywhere seems rather cumbersome and easy to forget.